setup/cluster/README.md

```diff
@@ -0,0 +1,228 @@
+
+## Requirements
+
+We assume you have a running Kubernetes cluster already. Since this tutorial works with the CLI, we furthermore assume you have the `docker`, `git`, `helm`, `kubectl`, `make` binaries installed and usable, i.e. having run `docker login` and having switched to the appropriate `kubectl` context.
+
+> This tutorial was tested on a machine running Ubuntu 20.04. It worked an Azure Kubernetes Service (AKS) cluster, an Elastic Kubernetes Service (EKS) cluster based on EC2 instances and with a local minikube cluster.
+
+## Starting and testing a deployment of Connaisseur
+
+### 1. Set up environment
+
+First off, we'll export the variables used in this tutorial to allow you to immediately use your own names/URLs instead of relying on our default names. Below, substitute your own values as appropriate:
+
+```bash
+REGISTRY_URL=docker.io
+REPOSITORY_NAME=testingconny
+IMAGE_NAME=testimage
+IMAGE_PATH=$(echo ${REGISTRY_URL}/${REPOSITORY_NAME}/${IMAGE_NAME})
+
+NOTARY_URL=notary.docker.io
+NOTARY_USER=<YOUR-NOTARY-USERNAME/DOCKER-HUB-ID>
+NOTARY_PASSWORD=<YOUR-NOTARY-PASSWORD>
+
+git clone https://github.com/sse-secure-systems/connaisseur.git
+cd connaisseur
+```
+
+### 2. (Optional) Prepare test images
+
+Go ahead and build both an unsigned and a signed image:
+
+```bash
+cd setup
+docker build -f Dockerfile.unsigned -t ${IMAGE_PATH}:unsigned .
+docker push ${IMAGE_PATH}:unsigned
+docker build -f Dockerfile.signed -t ${IMAGE_PATH}:signed .
+DOCKER_CONTENT_TRUST=1 docker push ${IMAGE_PATH}:signed
+cd -
+```
+
+During the second push, you will be asked to create a root key password and a repository key password (unless you already signed images or pushed signed images to the repository, respectively).
+
+### 3. Configure Connaisseur
+
+Before you can finally set up Connaisseur, you will need to configure its connection to the notary server provided by Docker Hub (or your notary server of choice).
+
+> At this point, Connaisseur only supports BasicAuth for connecting to the notary.
+
+Below is a list of commands to programmatically do the changes to the `value.yaml` file in the `helm` folder. The openssl command will take a password as input, so you won't be able to copy+paste past it. However, you can also do these changes manually if you want. They are comprised of setting the notary server, notary credentials and the root publiy key for verification. If you did not do step 2, you will need to enter your registry's root key from a (potentially) different source.
```

Comment:
```suggestion
Below is a list of commands to programmatically do the changes to the `values.yaml` file in the `helm` folder. The openssl command will take a password as input, so you won't be able to copy+paste past it. However, you can also do these changes manually if you want. They are comprised of setting the notary server, notary credentials and the root publiy key for verification. If you did not do step 2, you will need to enter your registry's root key from a (potentially) different source.
```
I am not certain what you mean that I will not be able to 'copy paste past it'. Does that mean if I copy the full code and run it in my shell that it fails bc it does not wait for the pw for OpenSSL? If so, I would split it in 2 code sections to avoid confusion. Maybe mention afterwards that a `git diff` will help to understand the changes implemented at this point.

Reply:
> Does that mean if I copy the full code and run it in my shell that it fails bc it does not wait for the pw for OpenSSL?
yes
Can change it